doc/gov/chapters/chapter04.md

```diff
@@ -4,66 +4,114 @@
 
+For each Workstream, following leadership positions exist:
+- Lead.
+- Up to two Co-leads. 
+
 <a name="4.2.1"></a>
 ### 4.2.1 Length of Terms
 
+The length of term for all CNTT leadership roles is **12** calender months.
+
 <a name="4.2.2"></a>
 ### 4.2.2 Representation
 
+Any Workstream Leads or Co-Leads positions can be held by any Participant.
+
+<a name="4.2.3"></a>
+### 4.2.3 Nomination
+
+- Any individual can nominate themselves or another participant for a given lead/co-lead position (the nominee must accept nomination if was nominate by someone else) as long as they agree to CNTT ToR.
+- An individual can self-nominate themselves or another participant to multiple positions.
+
+<a name="4.2.4"></a>
+### 4.2.4 Selection
+
+There will be a nomination window during which individuals can nominate others or nominate themselves for a position. The start and end date of the nomination window will be determined by Technical or Governance Committee Leads(depending on the nature of the Workstream).
+
+During the nomination period, Participants can send their nominations to LFN Staff who will help administering the process. The details and email addresses of LFN staff will be clarified when the process is started.
+
+Once the nomination period expires for a particular Workstream, the selection process will start as follows:
+
+- Selection process for Leads positions and Co-Leads will occur separately.
+- No voting will be required if number of nominations received for a particular position is less than or equal to the number of available vacancies. 
+- Online Voting process will take place to select candidate(s) from the nominee list for any given position.
+  - The exact tool used for voting will be determined by the LFN staff who will help administering the process and will depends on the approved tool at that point of time.
+- Only Voting representatives can vote (from the up to date voting representative list).
+- Candidate(s) who receives the highest number of votes win.
+
+<a name="4.2.5"></a>
+### 4.2.5 Interim Leads/Co-Leads
+
+- In case a Lead steps down, an interim person (preferably from the existing Co-leads) will be assigned by Technical Steering/Governance Committee to take over position duties until next available election.
+
+- In case a Co-Lead steps down, an interim person will be assigned by Technical Steering/Governance Committee to take over position duties until next available election.
+
+- For new Work Streams, a new Lead/Co-Leads will be assigned by Technical Steering/Governance to be interim until next available election period.
+
 <a name="4.3"></a>
 ## 4.3 Technical Steering Committee Lead/Co-Lead Nomination and Selection
 
-<a name="4.3.1"></a>
-### 4.3.1 Length of Terms
-
-<a name="4.3.2"></a>
-### 4.3.2 Representation
+The same terms, process of nomination and selection described in [**4.2**](#4.2) must be applied to technical steering committee leads and co-leads positions.
```

Comment:
Suggest change "... must be applied  .." to "... apply..."

Later on also.

Reply:
Fixed in https://github.com/cntt-n/CNTT/pull/896/commits/c836b429d699b8a973b65caacd1dfafddd88ae51

Reply:
feel free Pankaj to edit directly such kind of very minor editing.